src/node-tree.js

```diff
@@ -220,17 +220,21 @@ const processTextBlock = textBlock => {
  */
 export class NodeTree {
   constructor(richText) {
-    this.children = richText.reduce((children, currentBlock) => {
+    this.richText = richText
+  }
+
```

Comment:
Hmm... sorry I thought it would be `this.children = this.getChildren()` and getChildren() can be private